### PR TITLE
Workbench Integration

### PR DESCRIPTION
**Pull request by crynobone**
This PR introduces Workbench Integration where you can preview a demo Nova application directly inside your package using `composer run serve`.

### CODE REVIEW

File: testbench.yaml [hunk 1]
```diff
@@ -1,8 +1,25 @@
 providers:
-  - Oneduo\NovaFileManager\ToolServiceProvider
-  - Inertia\ServiceProvider
-  - Laravel\Nova\NovaServiceProvider
-  - Laravel\Nova\NovaApplicationServiceProvider
   - Laravel\Nova\NovaCoreServiceProvider
-  - NovaKit\NovaPackagesTool\LaravelServiceProvider
-  - Spatie\LaravelRay\RayServiceProvider
+  - Oneduo\NovaFileManager\ToolServiceProvider
+  - Workbench\App\Providers\NovaServiceProvider
```
Comment (crynobone) [inline testbench.yaml:4]: `post-autoload-dump` already runs `testbench package:discover` so manually adding the above service provider isn't no longer required.

**Comment by mikaelpopowicz** [comment]
Hi @crynobone 

Thank you! 🚀

**Comment by mikaelpopowicz** [comment]
Hi @crynobone 

I just noticed that the `serve` composer script timeout after 300 seconds, which is the composer's default process timeout.

The `Orchestra\Testbench\Foundation\Console\ServeCommand` may disable the process timeout if `composer/composer` is installed.

```php
if (
    class_exists(ComposerConfig::class, false)
    && method_exists(ComposerConfig::class, 'disableProcessTimeout') // @phpstan-ignore-line
) {
    ComposerConfig::disableProcessTimeout();
}
```

We may disable the process timeout directly in the scripts section as described in the [composer config documentation](https://getcomposer.org/doc/06-config.md#process-timeout)

```diff.json
{
  "scripts": {
    "post-autoload-dump": [
      "@prepare",
      "@dusk:install-chromedriver"
    ],
    "clear": "@php vendor/bin/testbench package:purge-skeleton --ansi",
    "prepare": "@php vendor/bin/testbench package:discover --ansi",
    "dusk:install-chromedriver": "@php vendor/bin/dusk-updater detect --auto-update --ansi",
    "build": "@php vendor/bin/testbench workbench:build --ansi",
    "serve": [
+    "Composer\\Config::disableProcessTimeout",
      "@build",
      "@php vendor/bin/testbench serve"
    ]
}
```

**Comment by crynobone** [comment]
https://github.com/orchestral/testbench-core/blob/8.x/src/Foundation/Console/ServeCommand.php#L23-L28

**Pull request merged**